test/compilable/test8707.sh

```diff
@@ -0,0 +1,15 @@
+if [ "${OS}" != "linux" ]; then
+    echo "Skipping void initialization test on $OS."
+fi
```

Comment:
missing `exit 0` here?

Reply:
No need - that's the default.

Reply:
You are not skipping the rest of the code here.